src/ProjectOrigin.WalletSystem.IntegrationTests/MessageBroker/RabbitMqTests.cs

```diff
@@ -0,0 +1,29 @@
+using ProjectOrigin.WalletSystem.IntegrationTests.TestClassFixtures;
+using ProjectOrigin.WalletSystem.Server;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ProjectOrigin.WalletSystem.IntegrationTests.MessageBroker;
+
+public class RabbitMqTests : WalletSystemTestsBase, IClassFixture<RabbitMqFixture>
```

Comment:
Should this be removed in this PR?

Reply:
It still verifies that the Host server can start with the configuration, a good test could be to also send an event, and wait for it be processed. This can be added later.